lib/commands/proxy-helper.js

```diff
@@ -44,7 +89,7 @@ helpers.proxyCommand = async function (endpoint, method, body, isSessionCommand
   if (!cmdName) {
     // this should never happen except when adding new routes
     cmdName = 'Unknown'; // just for logging purposes below
-    log.warn(`Proxying to WDA with an unknown route: ${method}:${endpoint}`);
+    log.warn(`Proxying to WDA with an unknown route: ${method} ${endpoint}`);
```

Comment:
if this should never happen, should we throw instead of warn? or at least log with error level letting people know to contact the devs?

Reply:
Well... it's not that pressing. We definitely don't want to throw an error, and I don't really want to alarm anyone with error messages. This largely gives something to search through Travis logs for, to make sure there is nothing left. 

The only issue that the route not being there causes is that you cannot tune the command timeout for it if there is no name associated with the route. It will fall back to the default.

Reply:
got it, makes sense